### PR TITLE
Update for hello_pull.txt

### PR DESCRIPTION
**Pull request by saba-ja**
This change will make the text more important

### CODE REVIEW

File: hello_pull.txt [hunk 1]
```diff
@@ -1 +1 @@
-Hello pull request tutorial
+This is important Hello pull request tutorial
```
Comment (saba-j) [inline hello_pull.txt:1]: This is not necessary.

Comment (saba-j) [reply hello_pull.txt:1]: Please provide more explanation

Comment (saba-ja) [reply hello_pull.txt:1]: In my opinion this can emphasize the importance of the Hello world!!

Comment (saba-j) [reply hello_pull.txt:1]: LGTM
(Looks Good To Me)

**Comment by saba-j** [comment]
Thank you for your contribution

**Pull request merged**